.browserslistrc

```diff
@@ -0,0 +1,2 @@
+last 2 versions
```

Comment:
This is now preferred instead of passing the `browsers` option to `ember-cli-autoprefixer`.